web/README.md

```diff
@@ -0,0 +1,242 @@
+# Rafiki Dashboard
+
+## How to run the app
+
+```
+yarn start
+```
+
+open https://localhost:3000/
+
+
+## How to setup Api End Point (environment parameters)
+
+```
+$ vim .env
+```
+
+```
+NODE_PATH=./src
+REACT_APP_WEBSITE_NAME="Rafiki DashBoard"
+REACT_APP_API_POINT_HOST=ncrs.d2.comp.nus.edu.sg
+REACT_APP_API_POINT_PORT=3000
+```
+
+## API Endpoints
```

Comment:
I'm thinking that this section is more appropriate in `rafiki/admin/` or as part of Rafiki's documentation as this is describing the Rafiki Admin's APIs. I will be moving this in the future.